pages/specification/migration/_index.md

```diff
@@ -5,76 +5,76 @@ section: docs
+| ----------------------- | ------------- | ------------------ | ------- | ------------------------------ |
+| `$anchor`               | Core          | 2019-09            | No      | No (updated in 2020)           |
+| `$comment`              | Core          | 07                 | Yes     | -                              |
+| `$id`                   | Core          | 06                 | No      | Replaced `id`                  |
+| `$defs`                 | Core          | 2019-09            | No      | Replaced `definitions`         |
+| `$dynamicAnchor`        | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`    |
+| `$dynamicRef`           | Core          | 2020-12            | No      | Replaced `$recursiveRef`       |
+| `$recursiveAnchor`      | Core          | 2019-09            | No      | Replaced by `$dynamicAnchor`   |
+| `$recursiveRef`         | Core          | 2019-09            | No      | Replaced by `$dynamicRef`      |
+| `$ref`                  | Core          | 2019-09            | No      | No                             |
+| `$schema`               | Core          | 02                 | No      | No                             |
+| `$vocabualry`           | Core          | 2019-09            | No      | No                             |
+| `additionalItems`       | Core          | 03                 | No      | No                             |
+| `allOf`                 | Core          | 04                 | No      | No                             |
+| `anyOf`                 | Core          | 04                 | No      | No                             |
+| `const`                 | Validation    | 06                 | No      | No                             |
+| `contains`              | Core          | 06                 | No      | No                             |
+| `contentEncoding`       | Validation    | 07                 | No      | No                             |
+| `contentMediaType`      | Validation    | 07                 | No      | No                             |
+| `contentSchema`         | Validation    | 2019-09            | No      | No                             |
+| `definitions`           | Validation    | 04                 | No      | Replaced by` $defs`            |
+| `default`               | Validation    | 01                 | No      | No                             |
+| `dependencies`          | Core          | 03                 | No      | No                             |
+| `dependentRequired`     | Validation    | 2019-09            | No      | No                             |
+| `dependentSchemas`      | Core          | 2019-09            | No      | Replaced `dependencies`        |
+| `deprecated`            | Validation    | 2019-09            | No      | No                             |
+| `disallow`              | Validation    | 03                 | Yes     | No                             |
```

Comment:
`disallow` was effectively the inverse of `type`. That was make unnecessary when `not` was introduced and could inverse any statement, not just `type`. So, I'm not sure saying it was replaced by `not` really makes sense by itself without more explanation.

```suggestion
| `disallow`              | Core          | 01                 | 04      | Replaced by `not`              |
```